pep_sphinx_extensions/pep_processor/parsing/pep_banner_directive.py

```diff
@@ -125,3 +125,39 @@ class CanonicalTypingSpecBanner(PEPBanner):
     admonition_class = nodes.attention
 
     css_classes = ["canonical-typing-spec", "sticky-banner"]
+
+
+class AbandonedBanner(PEPBanner):
+    """Insert an admonition for abandoned PEPs."""
+
+    admonition_class = nodes.warning
+    admonition_pre_template = "{link_content}"
+    admonition_pre_text = "This PEP has been abandoned."
+    css_classes = ["sticky-banner", "deprecated", "abandoned"]
+
+
+class RejectedBanner(PEPBanner):
+    """Insert an admonition for rejected PEPs."""
+
+    admonition_class = nodes.warning
+    admonition_pre_template = "{link_content}"
+    admonition_pre_text = "This PEP has been rejected."
+    css_classes = ["sticky-banner", "deprecated", "rejected"]
+
+
+class SupersededBanner(PEPBanner):
+    """Insert an admonition for superseded PEPs."""
+
+    admonition_class = nodes.warning
+    admonition_pre_template = "{link_content}"
```

Comment:
```suggestion
    admonition_pre_template = "This PEP has been superseded by :pep:`{link_content}`."
```

Or if we did this (as originally intended by the parent directive), we could greatly simplify normal usage from

```rst
.. superseded:: This PEP has been superseded by :pep:`292`.
```

To just

```rst
.. superseded:: 292
```